itou/templates/apply/list_for_siae.html

```diff
@@ -35,6 +35,27 @@
     {% include "apply/includes/job_applications_filters/offcanvas.html" %}
     <section class="s-section">
         <div class="s-section__container container">
+            <div class="alert alert-important alert-dismissible-once d-none my-5" role="status">
+                <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Fermer"></button>
+                <div class="row">
+                    <div class="d-none d-md-inline col-md-auto">
+                        <img src="{% static_theme_images 'ico-bicro-important.svg' %}" alt="" height="80">
+                    </div>
+                    <div class="col-12 col-md px-md-0">
+                        <p class="mb-2">
+                            <strong>Amélioration de votre gestion des candidatures</strong>
+                        </p>
+                        <p class="mb-0">
+                            Exprimez-vous sur 5 propositions pour nous aider à identifier celles qui simplifieront votre gestion des candidatures. (Temps estimé : 5 minutes)
```

Comment:
```suggestion
                            Exprimez-vous sur 5 propositions pour nous aider à identifier celles qui simplifieront votre gestion des candidatures. (Temps estimé : 5 minutes)
```

Espace insécable pour ne pas avoir ça : 
![image](https://github.com/user-attachments/assets/db25e73c-12d3-42dd-be2b-a24dd930014d)